src/java.base/share/classes/sun/security/util/ECUtil.java

```diff
@@ -375,5 +381,14 @@ public static void validatePublicKey(ECPoint point, ECParameterSpec spec)
         }
     }
 
+    public static boolean isBrainpoolP512r1(ECParameterSpec name) {
```

Comment:
Please revert changes to this file (this method belongs in `NativeECUtil`).

Reply:
Updates to `ECUtil` are now reverted. All updates were moved to `NativeECUtil` class.